Source/PropertyTools.Wpf/DataGrid/DataGrid.Strategy.cs

```diff
@@ -201,6 +203,31 @@ public override void GenerateColumnDefinitions()
                     return;
                 }
 
+                var view = CollectionViewSource.GetDefaultView(this.ItemsSource);

```

Comment:
will you also move the new code to a new "operator" class. I think this is better than to add a long block with a `return` in the end, even if some of the code will be duplicated.